pages/analysis.md

```diff
@@ -1,15 +1,132 @@
+{% include nav.html %} {% include css.html %}
+
 # Analysis
+---
+	
+The visualization on this page represents the documented interference
```

Comment:
As in the workflow page, you gotta watch your paragraph breaks. What you meant to be one paragraph (ending in "the market itself"), should all be on one line (meaning you don't hit the return button) according to Markdown. But you have it on 19 separate lines. Every other paragraph has this issue as well. Most of the time you don't notice it, but it can cause weird effects depending on the monitor size of the viewer. It's not a big deal, but you might want to consider fixing it. I have fixed the first paragraph below.

You would need to put all this on one line, and then remove the extra lines.

```suggestion
The visualization on this page represents the documented interference on the slave trade by privateers and pirates between 1699-1827. The primary geographical focus is in the Caribbean and West Indies region. These regions are where the most slave trade activity took place. The original hypothesis for this research is that privates and privateers had a significant effect on the slave trade. Pirates and privateers caused fiscal strains on the slave trade market and economically effected countries that participated in the slave trade. As per the documented information, the slave trade was extremely expansive that the incursion by privateers had little to no effect on the market itself. 
```